components/ResourceDetailsCard.tsx

```diff
@@ -15,18 +15,30 @@ const ResourceDetailsCard = (props: Props) => {
 
   if (!resource) return null;
 
+  // Preparing data
+
+  let image = "";
+  if (resource.images && resource.images.length > 0) {
+    const imageFile = resource.images[0];
+    image = `data:${imageFile.mimeType};base64,${imageFile.bin}`;
+  }
+
+  const type = resource.conformsTo?.name;
+
   return (
     <Card>
-      <div className="p-4 flex space-x-4">
-        <div className="w-20 h-20 rounded-md bg-gray-200" />
-        <div>
-          <Text as="h4" variant="bodyMd" fontWeight="bold">
+      <div className="p-4 flex flex-row items-center space-x-4">
+        <div className="w-20 h-20 rounded-md bg-gray-200 overflow-hidden flex items-center justify-center">
+          {image && <img src={image} alt="" className="w-full h-full object-cover" />}
+          {!image && <Icon color="subdued" source={DynamicSourceMinor} />}
+        </div>
+
+        <Stack vertical spacing="extraTight">
+          <Text as="h4" variant="headingMd" fontWeight="bold">
```

Comment:
can we have more semantic components here, like I imagine more a `<Heading>{text}</Heading>` or `MediumHeading` here than to remember the correct style name and the h4